helm/values.schema.json

```diff
@@ -192,7 +192,7 @@
       "type": "array",
       "items": {
         "type": "string",
-        "pattern": "^.*=.*$"
+        "pattern": "(^$|^.*=.*$)"
```

Comment:
hmm, this seems like an odd change, @brucegucode. Is this related to DB cluster restore from snapshot?

Reply:
this was not related to this pr, it comes from generating code with latest code-generator. https://github.com/aws-controllers-k8s/code-generator/commit/cb4bca850e1d1dd678d8ae9bd9d5fd537c06ca51

Reply:
Got it. :)